danger-spelling.gemspec

```diff
@@ -17,27 +17,33 @@ Gem::Specification.new do |spec|
   spec.files         = `git ls-files`.split($/)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/HamstringAssassin/danger-spelling/blob/8b5ef5e47ae075e09604a70ce13103f01902f481/danger-spelling.gemspec#L17"></span>Prefer <code>$INPUT_RECORD_SEPARATOR</code> or <code>$RS</code> from the stdlib 'English' module (don't forget to require it) over <code>$/</code>.</td>
    </tr>
  </tbody>
</table>